.circleci/config.yml

```diff
@@ -1,6 +1,29 @@
-version: 2
+version: 2.1
+
+parameters:
+  # This parameter is used to trigger the main workflow
+  trigger:
+    type: boolean
+    default: true
+
+  # A parameter per group
+  backend:
+    type: boolean
+    default: false
+  frontend:
+    type: boolean
+    default: false
 
 jobs:
+  trigger-workflows:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - run:
+          name: Trigger workflows
+          command: chmod +x .circleci/conditional-builds.sh && .circleci/conditional-builds.sh
```

Comment:
Could commit the file to git with the +x permissions added already?

Reply:
Good call! I'll give it a try.